### PR TITLE
add test & fix empty slice element bug

### PR DESCRIPTION
**Pull request by alexandrinaw**
Added a test, watched it fail, implemented the fix.

### CODE REVIEW

File: dbstore.go [hunk 1]
```diff
@@ -612,8 +614,10 @@ func (p *persister) save(ctx context.Context, ws *Worksheet) error {
 				})
 			}
 		}
-		if _, err := insertSliceElements.ExecContext(ctx); err != nil {
-			return err
+		if atLeastOneSliceElementExists {
```
Comment (cbelsole) [inline dbstore.go:617]: Annoyingly the insert builder does not expose `records` which we could check the `len()` of.

**Review by cbelsole — APPROVED** [review]
